Data/models/DataController.swift

```diff
@@ -146,14 +153,24 @@ public class DataController: NSObject {
             }
         }
     }
-}
-
-extension NSManagedObjectContext {
-    static var mainThreadContext: NSManagedObjectContext {
-        return DataController.shared.mainThreadContext
+    
+    public static func resetDatabase() {
```

Comment:
This should probably be in a more unit test environment, putting unit test functions in general classes can get pretty messy.

Reply:
I would need to make initializer at least `internal` then. If you prefer it this way I can change it.